src/core/server/uuid/manage_uuid.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import uuid from 'uuid';
+import { promisify } from 'util';
+import * as fs from 'fs';
+import { join } from 'path';
+import { take } from 'rxjs/operators';
+import { IConfigService } from '../config';
+import { PathConfigType } from '../path';
+import { HttpConfigType } from '../http';
+
+const FILE_ENCODING = 'utf8';
+const FILE_NAME = 'uuid';
+
+const readFile = promisify(fs.readFile);
+const writeFile = promisify(fs.writeFile);
+
+export async function manageInstanceUuid(configService: IConfigService): Promise<string> {
```

Comment:
```suggestion
export async function resolveInstanceUuid(configService: IConfigService): Promise<string> {
```
seems more descriptive than `manage`?

Reply:
Probably better. Wasn't that inspired with the name. Changed.